openapi/bitbucket/client.bal

```diff
@@ -37,6 +39,41 @@ public client class Client {
         http:Client httpEp = check new (serviceUrl, { auth: clientConfig.authConfig, secureSocket: secureSocketConfig });
         self.clientEp = httpEp;
     }
+    # Returns a list of all repositories
+    #
+    # + workspace - This can either be the workspace ID (slug) or the workspace UUID
+    # + role - Filters the result based on the authenticated user's role on each repository. member-returns repositories to which the user has explicit read access. contributor- returns repositories to which the user has explicit write access. admin- returns repositories to which the user has explicit administrator access. owner-returns all repositories owned by the current user
```

Comment:
Need to break this after 120 characters.
Check other places also

Reply:
@SanduDS It seems you are not using the latest tool to generate the code. We have patched it to fix these issues.